Engine/ac/overlay.cpp

```diff
@@ -138,6 +138,24 @@ ScriptOverlay* Overlay_CreateTextual(int x, int y, int width, int font, int colo
     return sco;
 }
 
+int Overlay_GetBlendMode(ScriptOverlay *scover) {
+    int ovri = find_overlay_of_type(scover->overlayId);
+    if (ovri < 0)
+        quit("!invalid overlay ID specified");
+
+    return screenover[ovri].blendMode;
+}
+
+void Overlay_SetBlendMode(ScriptOverlay *scover, int blendMode) {
+    int ovri = find_overlay_of_type(scover->overlayId);
+    if (ovri < 0)
+        quit("!invalid overlay ID specified");
+    if ((blendMode < 0) || (blendMode >= kNumBlendModes))
+        quitprintf("!SetBlendMode: invalid blend mode %d, supported modes are %d - %d", 0, kNumBlendModes - 1);
```

Comment:
You're missing blendMode in the variadic parameters, so the error is incorrectly displayed (same for the other instances)

Reply:
ah shoot.